wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -160,8 +160,8 @@ class IWearLogger::impl
     yarp::dev::IPreciselyTimed* iPreciselyTimed = nullptr;
     std::mutex loggerMutex;
     IWearLoggerSettings settings;
-    yarp::telemetry::experimental::BufferConfig bufferConfig;
-    yarp::telemetry::experimental::BufferManager<double> bufferManager;
+    robometry::BufferConfig bufferConfig;
+    robometry::BufferManager<double> bufferManager;
```

Comment:
I would suggest removing the template

Reply:
Accordingly to https://github.com/robotology/robometry/pull/175 we should now be able to remove the template, am I right?

Reply:
Yes exactly, I removed it